### PR TITLE
Feature/merge all

### PR DESCRIPTION
**Pull request by wgr2017**
*Issue #, if available:*

*Description of changes:*
Merged all branches, updated README.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-nitro-enclave-blockchain-wallet.generated.sln
Comment (dpdornseifer) [inline aws-nitro-enclave-blockchain-wallet.generated.sln:1]: not required in repo

File: application/wireguard/enclave/kms/kmstool_enclave_cli
Comment (dpdornseifer) [inline application/wireguard/enclave/kms/kmstool_enclave_cli:1]: binary should be removed from repo

File: application/wireguard/enclave/kms/libnsm.so
Comment (dpdornseifer) [inline application/wireguard/enclave/kms/libnsm.so:1]: binary should be removed from repo. `.gitignore` should be extended to exclude all `kms` related build artifacts

File: application/eth1/enclave/netenclave/obj/project.nuget.cache
Comment (dpdornseifer) [inline application/eth1/enclave/netenclave/obj/project.nuget.cache:1]: merge issue happened / eth1/enclave files are missing `server.py, requirements.txt, Dockerfile`: https://github.com/aws-samples/aws-nitro-enclave-blockchain-wallet/tree/main/application/eth1/enclave

File: application/dotnet_sqs_integration/enclave/server.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        aws_access_key_id,
+        "--aws-secret-access-key",
+        aws_secret_access_key,
+        "--aws-session-token",
+        aws_session_token,
+        "--ciphertext",
+        ciphertext,
+    ]
+
+    print("subprocess args: {}".format(subprocess_args))
```
[the remark this answers is not in the document]
Comment (dpdornseifer) [reply application/dotnet_sqs_integration/enclave/server.py:36]: print statements inside enclave are not being exposed if debug is turned off

File: application/eth1/enclave/netenclave/obj/Debug/net6.0/netenclave.GeneratedMSBuildEditorConfig.editorconfig
Comment (dpdornseifer) [inline application/eth1/enclave/netenclave/obj/Debug/net6.0/netenclave.GeneratedMSBuildEditorConfig.editorconfig:1]: debug files should not be included in repo / local path exposed

File: application/eth1/enclave/netenclave/obj/netenclave.csproj.nuget.dgspec.json
Comment (dpdornseifer) [inline application/eth1/enclave/netenclave/obj/netenclave.csproj.nuget.dgspec.json:1]: debug files should not be included in repo / local path exposed / valid for all files in directory

File: application/https_web_server/enclave/secure_server.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        self.send_response(200)
+        self.send_header("Content-Type", "text/html")
+        self.send_header("Content-Length", len(DUMMY_RESPONSE))
+        self.end_headers()
+        self.wfile.write(DUMMY_RESPONSE)
+
+ssl_context = SSLContext(PROTOCOL_TLS_SERVER)
+ssl_context.load_cert_chain("/app/certs/cert.pem", "/app/certs/key.pem")
+server = HTTPServer(("", 9001), MyHandler)
+server.socket = ssl_context.wrap_socket(server.socket, server_side=True)
```
[the remark this answers is not in the document]
Comment (dpdornseifer) [reply application/https_web_server/enclave/secure_server.py:33]: @wgr2017 ssl version should be fixed to min version >=v1.2

File: application/rds_integration/enclave/requirements.txt
Comment (dpdornseifer) [inline application/rds_integration/enclave/requirements.txt:1]: psycopg2-binary version pin missing

File: application/rds_integration/enclave/server.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        aws_access_key_id,
+        "--aws-secret-access-key",
+        aws_secret_access_key,
+        "--aws-session-token",
+        aws_session_token,
+        "--ciphertext",
+        ciphertext,
+    ]
+
+    print("subprocess args: {}".format(subprocess_args))
```
[the remark this answers is not in the document]
Comment (dpdornseifer) [reply application/rds_integration/enclave/server.py:40]: code running inside enclave with no logging capabilities if debug mode turned off

File: application/socat/enclave/run_socat.sh
Comment (dpdornseifer) [inline application/socat/enclave/run_socat.sh:1]: `iperf3` standard invoke should be commented out

File: README.md
Comment (dpdornseifer) [inline README.md:1]: we should assume `eth1` as default application type and provide a list of all the other options on the central readme.md

**Review by dpdornseifer — CHANGES_REQUESTED** [review]
PR looks good to me - a few minor cleanup tasks required

File: application/rds_integration/enclave/requirements.txt
Comment (wgr2017) [reply application/rds_integration/enclave/requirements.txt:1]: this seems to be missing in the original repo as well

File: README.md
Comment (wgr2017) [reply README.md:1]: I believe that is what the current setup is. The central readme.md instructs about the default application type `eth1`, and has links to other readme files.

File: aws-nitro-enclave-blockchain-wallet.generated.sln
Comment (wgr2017) [reply aws-nitro-enclave-blockchain-wallet.generated.sln:1]: done

**Review by dpdornseifer — APPROVED** [review]
lgtm

**Pull request merged**